app/app.py

```diff
@@ -14,19 +14,24 @@
 
 def create_app(test_config=None):
 
+    # app = Flask(__name__,
+    #         static_url_path='/static', 
+    #         static_folder='../frontend/static',
+    #         template_folder='../frontend/templates')
+
     app = Flask(__name__,
-            static_url_path='/static', 
-            static_folder='../frontend/static',
-            template_folder='../frontend/templates')
+        static_url_path='/static', 
+        static_folder='/app/frontend/static',
```

Comment:
There's two options here: either we can do something using `__file__` to get the relative path, since this won't change. We could also setup some environment variables.